bootstrap-agent-api/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "bootstrap-agent-api"
```

Comment:
Should this live under sled-agent, just like nexus-internal-api lives under nexus? I think the actual sled-agent API would live under sled-agent.

Reply:
I kinda think that makes sense, though I acknowledge "clearly either way could work".

Just having the directory within `sled-agent` makes it more clear to me, IMO, that the bootstrap agent is software running within the sled agent process.

Reply:
Done.